src/bus/peer.c

```diff
@@ -651,9 +651,12 @@ int peer_queue_call(PolicySnapshot *sender_policy, NameSet *sender_names, MatchR
                                        message->metadata.fields.path,
                                        message->header->type);
         if (r) {
-                if (r == POLICY_E_ACCESS_DENIED) {
+                if (r == POLICY_E_ACCESS_DENIED || r == POLICY_E_SELINUX_ACCESS_DENIED) {
                         log_append_here(receiver->bus->log, LOG_WARNING, 0);
-                        r = bus_log_commit_policy_send(receiver->bus, sender_id, receiver->id, sender_names, &receiver_names, message);
+                        if (r == POLICY_E_ACCESS_DENIED)
+                                r = bus_log_commit_policy_send(receiver->bus, sender_id, receiver->id, sender_names, &receiver_names, message);
+                        else
+                                r = bus_log_commit_policy_selinux_send(receiver->bus, sender_policy->seclabel, receiver->policy->seclabel, sender_id, receiver->id, sender_names, &receiver_names, message);
```

Comment:
Thinking about it more: Can't we make the "policy-type" argument an enum? Rather htan making it separate function names... We can still store strings in the log to avoid log-ABI. But the internal API now looks weird.

Reply:
Done.